src/clr-angular/forms/datepicker/providers/date-navigation.service.spec.ts

```diff
@@ -0,0 +1,233 @@
+
+            it("provides a function to update the Calendar Month", () => {
+                initalizeCalendar(new DayModel(2015, 1, 1));
+                expect(dateNavigationService.displayedCalendar.month).toBe(1);
+
+                dateNavigationService.changeMonth(2);
+                expect(dateNavigationService.displayedCalendar.month).toBe(2);
+            });
+
+            it("provides a function to update the Calendar Year", () => {
+                initalizeCalendar(new DayModel(2016, 1, 1));
+                expect(dateNavigationService.displayedCalendar.year).toBe(2016);
+
+                dateNavigationService.changeYear(2018);
+                expect(dateNavigationService.displayedCalendar.year).toBe(2018);
+            });
+
+            it("provides a function to move to the next month", () => {
+                initalizeCalendar(new DayModel(2016, 11, 1));
+
+                expect(dateNavigationService.displayedCalendar.year).toBe(2016);
+                expect(dateNavigationService.displayedCalendar.month).toBe(11);
+
+                dateNavigationService.moveToNextMonth();
+                expect(dateNavigationService.displayedCalendar.year).toBe(2017);
+                expect(dateNavigationService.displayedCalendar.month).toBe(0);
+
+                dateNavigationService.moveToNextMonth();
+                expect(dateNavigationService.displayedCalendar.year).toBe(2017);
+                expect(dateNavigationService.displayedCalendar.month).toBe(1);
+            });
+
+            it("provides a function to move to the previous month", () => {
+                initalizeCalendar(new DayModel(2017, 0, 1));
+
+                expect(dateNavigationService.displayedCalendar.year).toBe(2017);
+                expect(dateNavigationService.displayedCalendar.month).toBe(0);
+
+                dateNavigationService.moveToPreviousMonth();
+                expect(dateNavigationService.displayedCalendar.year).toBe(2016);
+                expect(dateNavigationService.displayedCalendar.month).toBe(11);
+
+                dateNavigationService.moveToPreviousMonth();
+                expect(dateNavigationService.displayedCalendar.year).toBe(2016);
+                expect(dateNavigationService.displayedCalendar.month).toBe(10);
+            });
+
+            it("provides a function to move to the current year and month", () => {
+                initalizeCalendar(new DayModel(2017, 0, 1));
+                const date: Date = new Date();
+
+                expect(dateNavigationService.displayedCalendar.year).toBe(2017);
+                expect(dateNavigationService.displayedCalendar.month).toBe(0);
+
+                dateNavigationService.moveToCurrentMonth();
+                expect(dateNavigationService.displayedCalendar.year).toBe(date.getFullYear());
+                expect(dateNavigationService.displayedCalendar.month).toBe(date.getMonth());
+            });
+        });
+
+        describe("Handling Keyboard Events", () => {
```

Comment:
That describe doesn't make sense anymore.

Reply:
You are right. Moved the tests but didn't change the description. 🤦‍♂️